### ISSUE TITLE
Betterdiscord not starting with discord

### ISSUE THREAD

**Issue by LordMZTE** [bb010g/betterdiscordctl#100]
When starting discord, BetterDiscord doesn't launch with it.

## Logs
```
~
❯ betterdiscordctl install -v -v
VV: Scanning /opt
VV: Trying flavor ''
V: Using Discord at /opt/discord
V: Found modules in /home/lordmzte/.config/discord/0.0.14/modules
V: Injecting into index.js...
VV: Generating fresh bd_map number...
V: Linking BetterDiscord injector...
Installed. (Restart Discord if necessary.)

~
❯ betterdiscordctl status -v -v
VV: Scanning /opt
VV: Trying flavor ''
V: Using Discord at /opt/discord
V: Found modules in /home/lordmzte/.config/discord/0.0.14/modules
Discord: /opt/discord
Modules: /home/lordmzte/.config/discord/0.0.14/modules
Index modified: yes
Linked injector directory: /home/lordmzte/.local/share/betterdiscordctl/bd/0
Linked injector repository: https://github.com/rauenzi/BetterDiscordApp

~
❯ discord
Discord 0.0.14
(node:754618) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Starting app.
Starting updater.
[Modules] Modules initializing
[Modules] Distribution: remote
[Modules] Host updates: enabled
[Modules] Module updates: enabled
[Modules] Module install path: /home/lordmzte/.config/discord/0.0.14/modules
[Modules] Module installed file path: /home/lordmzte/.config/discord/0.0.14/modules/installed.json
[Modules] Module download path: /home/lordmzte/.config/discord/0.0.14/modules/pending
[Modules] No updates to install
[Modules] Checking for host updates.
[Modules] Host is up to date.
[Modules] Checking for module updates at https://discord.com/api/modules/stable/versions.json
[Modules] No module updates available.
```

I installed discord using [this package](https://archlinux.org/packages/community/x86_64/discord/)

![](https://i.imgur.com/LlSO9IV.png)

**Comment by LucentW** [comment]
Unfortunately it's a known bug, see issues BetterDiscordApp#598 and BetterDiscordApp#548.
Track those issues to know when it gets fixed.

**Comment by LordMZTE** [comment]
Alright, thanks for the clarification.

**Comment by bb010g** [comment]
This will be resolved with #120.
